website/docs/r/repository_tag_protection.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
New resources must be linked in the [github.erb](https://github.com/integrations/terraform-provider-github/blob/main/website/github.erb) file to be accessible on the Terraform website. 

Would you mind adding a link section there in alphabetical order?